src/nupic/proto/TemporalMemoryV1.capnp

```diff
@@ -0,0 +1,126 @@
+@0xa64ad1d724c51d4c;
+
+# TODO: Use absolute path
+using import "RandomProto.capnp".RandomProto;
+
+# Next ID: ???
+struct TemporalMemoryV1Proto {
+  version @0 :UInt16;
```

Comment:
I don't think we need a version. I can imagine some cases that we might but until then I think we should leave it out.